app/scripts/lib/transaction/smart-transactions.ts

```diff
@@ -313,7 +313,9 @@ class SmartTransactionHook {
       signedTransactions,
       signedCanceledTransactions: [],
       txParams: this.#txParams,
-      transactionMeta: this.#transactionMeta,
+      // TODO: Replace `any` with type - version mismatch between smart-transactions-controller and transaction-controller breaking type safety
```

Comment:
Any idea what the type incompatibilities are here? The SmartTransactionsController is using types from v34.0.0 of the TransactionController, and I didn't see any breaking changes in the changelog that might explain this.

Reply:
Ah, looks like it was this: https://github.com/MetaMask/core/pull/4714

The additional two types aren't expected by the SmartTransactionsController. It looks like the SmartTransactionsController doesn't have any matchers for this property, so it should not cause any problems there.